marvinbot/models.py

```diff
@@ -1,11 +1,24 @@
 from uuid import uuid5, NAMESPACE_X500
+from passlib.context import CryptContext
 from marvinbot.utils import localized_date
 from marvinbot.defaults import (
-    USER_ROLES, DEFAULT_ROLE, ADMIN_ROLE, OWNER_ROLE,
-    POWER_USERS
+    DEFAULT_ROLE, ADMIN_ROLE, OWNER_ROLE,
+    POWER_USERS, RoleType
 )
+
 import mongoengine
 
+bot_security_context = CryptContext(
+    schemes=["bcrypt", ],
+    default="bcrypt",)
+
+
+def hash_password(pwd):
+    return bot_security_context.encrypt(pwd)
```

Comment:
pwd+salt maybe?

Reply:
That `encrypt` method automagically handles it.

https://passlib.readthedocs.io/en/stable/faq.html